.github/workflows/assemble.yml

```diff
@@ -30,11 +30,8 @@ jobs:
       - name: Setup docker (missing on MacOS)
         id: setup_docker
         if: runner.os == 'macos'
-        uses: douglascamata/setup-docker-macos-action@main
-        continue-on-error: true
-        with:
-          upgrade-qemu: true
-          colima:  v0.6.8
+        run: |
+          exit 0;
```

Comment:
@dblock @jed326 the quick fixes I've tried did not succeed :(, making MacOS `assemble` a noop for now and will work through the day to fix it

Reply:
thanks @reta for helping here! debugging github actions is hard

Reply:
Sounds good, open an issue please so we don't lose it?

Reply:
:100:

Reply:
https://github.com/opensearch-project/OpenSearch/issues/15743